src/settings-ui/Settings.UI/Views/VideoConference.xaml.cs

```diff
@@ -19,17 +18,29 @@ public sealed partial class VideoConferencePage : Page, IRefreshablePage
 
         private static async Task<string> PickFileDialog()
```

Comment:
let's remove this `async` and return proper `openFileName.File`. This function is only called for this, shouldn't be a big change to make it work.

Reply:
done